packages/block-library/src/cover/editor.scss

```diff
@@ -1,13 +1,6 @@
 .wp-block-cover-image,
 .wp-block-cover {
-	.editor-rich-text__tinymce[data-is-empty="true"]::before {
```

Comment:
`data-is-empty` was removed in #1485 (thanks @aduth), so I'm removing the styles as they're most likely very outdated and haven't worked in 1.5 years.